packages/lesswrong/components/posts/PostsList2.jsx

```diff
@@ -40,23 +40,22 @@ const PostsList2 = ({ results, loading, count, totalCount, loadMore, networkStat
 
   const limit = terms.limit || 10
 
-  // We don't actually know if there are more posts here, 
+  // We don't actually know if there are more posts here,
   // but if this condition fails to meet we know that there definitely are no more posts
-  const maybeMorePosts = !!(results && results.length && (results.length >= limit))  
-                                                                                     
+  const maybeMorePosts = !!(results && results.length && (results.length >= limit))
```

Comment:
The limit used here is the first-page size, and doesn't update when you click load more. If `withMulti` is passing a prop that reflects the actual limit that was used, then switching it to use that would make the load more disappear when you got to the end of the list, which would be nice.